net/openssh/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openssh
-PKG_VERSION:=8.3p1
+PKG_VERSION:=8.4p1
 PKG_RELEASE:=2
```

Comment:
You should probably reset PKG_RELEASE to 1

Reply:
OK, changed.